src/scripts/object.search.js

```diff
@@ -173,27 +175,40 @@ module.exports = (robot, res) => {
 					robot.logger.debug(
 						`${TAG}: Downloading ${objectDetails.containerName} container and ${objectDetails.objectName} object for search command.`
 					);
-					downloadPromises.push(storage.getObject(objectDetails.containerName, objectDetails.objectName));
+					downloadPromises.push(storage.getObject(objectDetails.containerName, objectDetails.objectName).catch((err) => {
```

Comment:
Nice trick to avoid the fail-fast behavior of Promise.all.  I learned something new, thx!